include/anyrpc/xml/xmlwriter.h

```diff
@@ -42,8 +42,8 @@ class ANYRPC_API XmlWriter : public Handler
     //! Make the text more readable with indentation and linefeeds
     void SetPretty(bool pretty=true) { pretty_ = pretty; }
     //! Set the double format method. The default is no exponents for xmlrpc spec compatibility.
-    void SetScientificPrecision(unsigned precision=18)
-        { precision_ = std::min( 32u, precision_); }
+    void SetScientificPrecision(unsigned precision=32)
```

Comment:
Thanks for the bug fix on this.  Visual Studio and gcc didn't give a warning and it obviously didn't get properly tested.  I'd like to keep the default to 18 since that should be sufficient for double precision numbers.